docs/extras/building_vtk.rst

```diff
@@ -237,7 +243,7 @@ Where ``build_wheels.sh`` is:
           -DVTK_WHEEL_BUILD=ON \
           -DVTK_PYTHON_VERSION=3 \
           -DVTK_WRAP_PYTHON=ON \
-          -DVTK_OPENGL_HAS_EGL=True \
+          -DVTK_OPENGL_HAS_EGL=False \
```

Comment:
This is less clear-cut so I don't want to push it, but this code block has the EGL `yum install` command on top and the `/io/wheels_egl` path at the bottom. Might be worth changing too to be consistent with the non-EGL default.

Reply:
Another copy/pasta mistake.  Thanks for catching it.